include/tvm/target/target.h

```diff
@@ -168,5 +175,11 @@ class Target : public ObjectRef {
   TVM_DLL void ExitWithScope();
 };
 
+using TargetsMap = Map<Integer, Target>;
+
+TVM_DLL void RefreshHost(Target*, Target*);
+TVM_DLL void RefreshHost(TargetsMap*, Target*);
+TVM_DLL void RefreshHost(Map<Target, IRModule>*, Target*);
```

Comment:
I found that I cannot move the part of the helper function into the Target class because it requires a `Map<Target, ObjectRef>` type which cannot be compiled correctly. Therefore, I would put the functions outside the class for now.